### ISSUE TITLE
WidgetControl - Transparent background in HTML widget image replaced by white background.

### ISSUE THREAD

**Issue by flanabode** [jupyter-widgets/ipyleaflet#331]
I've noticed when trying to use WidgetControl to place an HTML widget containing an image with a transparent background on the map, a white background is created instead. Is there a way to preserve the transparent background the way that is native in FloatImage in Folium?

**Comment by martinRenou** [comment]
I don't think it is possible. If what you want is an equivalent of Folium's FloatImage, I guess we could add a new layer type in ipyleaflet.
If you feel like opening a PR please do! :)

**Comment by flanabode** [comment]
Thanks, @martinRenou - unfortunately I think pulling this together myself is a bit beyond my ken. I hope someone on your team can look into it in the future. Thanks again for your time!
